schema/omega-services.async.api.yaml

```diff
@@ -313,30 +250,150 @@ components:
           items:
             $ref: '#/components/schemas/separatedMaterial'
         creatorIDs:
-          description: |
-            Identifies the organisation(s) or individual(s) responsible for
-            creating, accumulating or maintaining the records being described.
-            Incorporates the authority controlled corporate or personal name(s)
-            of the creator(s) of the records.
+          $ref: "#/components/schemas/creatorIDs"
+
+    updateEditSetRecord:
+      type: object
+      description: Properties to update for an edit set record
+      properties:
+        description:
+          $ref: "#/components/schemas/description"
+        coveringDates:
+          $ref: "#/components/schemas/coveringDates"
+        formerReferenceDepartment:
+          $ref: "#/components/schemas/formerReferenceDepartment"
+        formerReferencePro:
+          $ref: "#/components/schemas/formerReferencePro"
+        startDate:
+          $ref: "#/components/schemas/start"
```

Comment:
I think this should be `$ref: "#/components/schemas/startDate"`

Reply:
Thanks, changed.

I wonder if there are tools to validate the schema itself, other than loading it into asyncapi studio. 
This repository might contain something useful:
https://github.com/asyncapi/spec-json-schemas

Reply:
Interesting thought! I have put a ticket into the backlog for some further investigation: https://national-archives.atlassian.net/browse/PACT-729